ledger/sandbox-on-x/src/main/scala/com/daml/ledger/sandbox/domain/Rejection.scala

```diff
@@ -64,6 +64,14 @@ private[sandbox] object Rejection {
       .rpcStatus(completionInfo.submissionId)
   }
 
+  final case class OffsetDeduplicationPeriodUnsupported(completionInfo: CompletionInfo)(implicit
+      contextualizedErrorLogger: ContextualizedErrorLogger
+  ) extends Rejection {
+    override def toStatus: Status = LedgerApiErrors.UnsupportedFeature
+      .Reject("offset-type deduplication period")
```

Comment:
I'd wager that the error message is quite hard to understand, as "offset-type deduplication period" is not really an operation. I'd expect there to be a verb or a sentence for describing an operation.

Reply:
rephrased